scripts/restore_backup.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script for backup restoration on backup migration server.
+
+This script should be run from the oppia root folder:
+
+    python -m scripts.restore_backup
+
+If the status of a backup restoration is to be checked, run the script as:
+
+    python -m scripts.restore_backup --check_status
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import os
+import sys
+
+import python_utils
+
+from . import common
+
+GCLOUD_PATH = os.path.join(
+    '..', 'oppia_tools', 'google-cloud-sdk-251.0.0', 'google-cloud-sdk',
+    'bin', 'gcloud')
+
+CURR_DIR = os.path.abspath(os.getcwd())
+OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
```

Comment:
Should `..` and `CURR_DIR` be switched?

Reply:
No, `oppia_tools` is in parent directory of `oppia` folder (`oppia/../oppia_tools`).